_posts/fr/newsletters/2022-09-21-newsletter.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'Bitcoin Optech Newsletter #218'
+permalink: /fr/newsletters/2022/09/21/
+name: 2022-09-21-newsletter-fr
+slug: 2022-09-21-newsletter-fr
+type: newsletter-fr
+layout: newsletter-fr
+lang: fr
+---
+Cette semaine la newsletter résume une discussion à propos de l'utilisation
+de `SIGHASH_ANYPREVOUT` pour reproduire des aspects des chaines dérivées.
+Sont inclus aussi nos sections habituelles abordant les derniers changements
```

Comment:
Suggest to remove trailing space on this line

Reply:
Done